### PR TITLE
fix: use string type, replace commas

### PR DESCRIPTION
**Pull request by jpangelle**
### Description

This pull request changes the type of `tokenAmount` from `number` to `string` and where necessary replaces the commas with whitespace.

### CODE REVIEW

File: src/Components/Pages/FormikContextElements/Fees.tsx [hunk 1]
```diff
@@ -35,8 +35,11 @@ const FeesFormikWrapped: React.FC<IFeesFormikWrapped> = ({
           <Typography component="p">Transfer Amount:</Typography>
           <Typography component="p">
             {Number(
-              (values as Record<string, string | number>)[amountFormikName],
-            )?.toFixed(3)}{' '}
+              (values as Record<string, string>)[amountFormikName].replaceAll(
+                ',',
```
Comment (vedhavyas) [inline src/Components/Pages/FormikContextElements/Fees.tsx:39]: do we already take care of spaces and other chars as well ?

**Review by onnovisser — APPROVED** [review]
Looks good to me, though I'm now a bit worried about people from countries where `.` separates 1000s and `,` separates decimals. Were they to enter `1.000,00` the result won't be what they expect.

**Comment by jpangelle** [comment]
> Looks good to me, though I'm now a bit worried about people from countries where `.` separates 1000s and `,` separates decimals. Were they to enter `1.000,00` the result won't be what they expect.

Good point. Just tried and the UI shows `Input invalid` so we should be good:
image

File: src/Components/Pages/FormikContextElements/Fees.tsx (hunk 1 above)
Comment (jpangelle) [reply src/Components/Pages/FormikContextElements/Fees.tsx:39]: Yup. If there are any spaces or any non-decimal characters, the form will show `Input invalid` when the user tries to submit.

**Pull request merged**